class-simple-local-avatars.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace WPSL\SimpleLocalAvatars;
+
+use wpCloud\StatelessMedia\Compatibility;
+
+class SimpleLocalAvatars extends Compatibility {
+  protected $id = 'simple-local-avatars';
+  protected $title = 'Simple Local Avatars';
+  protected $constant = 'WP_STATELESS_COMPATIBILITY_SLA';
+  protected $description = 'Ensures compatibility with Simple Local Avatars plugin.';
+  protected $plugin_file = 'simple-local-avatars/simple-local-avatars.php';
+  protected $sm_mode_not_supported = ['stateless'];
+
+  /**
+   * Initialize compatibility module
+   *
+   * @param $sm
+   */
+  public function module_init($sm) {
+    // Only makes sense in CDN or Ephemeral modes
+    if (in_array($sm['mode'], ['cdn', 'ephemeral'])) {
```

Comment:
@balexey88 
Let's add `stateless` mode here

Reply:
This is done )